docs/build.md

```diff
@@ -2,7 +2,7 @@
 
 Dependency | Version | Required           | Runtime
 ---        | ---     | ---                | ---
-Kokkos     | 2.8.00  | :heavy_check_mark: | :heavy_check_mark:
+Kokkos     | 3.0.0   | :heavy_check_mark: | :heavy_check_mark:
```

Comment:
```suggestion
Kokkos     | 3.0.00   | :heavy_check_mark: | :heavy_check_mark:
```
Unfortunately I am not kidding...

Reply:
> Unfortunately I am not kidding...

I updated, but why?

Reply:
https://github.com/kokkos/kokkos/releases/tag/3.0.00

You go ahead and ask Christian